pg2kinesis/formatter.py

```diff
@@ -13,7 +14,7 @@
 # Final product of Formatter, a Change and the Change formatted.
 Message = namedtuple('Message', 'change, fmt_msg')
 
-COL_TYPE_VALUE_TEMPLATE_PAT = ur"{col_name}\[{col_type}\]:'?([\w\-]+)'?"
+COL_TYPE_VALUE_TEMPLATE_PAT = u"{col_name}\\[{col_type}\\]:'?([\\w\\-]+)'?"
```

Comment:
Let keep the `r` form (and drop the 'u' since 'ur' stings are not a thing) and add `from __future__ import unicode_literals` to the top like we already do in tests__main__.py